draft-ietf-ppm-dap-taskprov.md

```diff
@@ -301,6 +311,34 @@ unimplemented VDAF).
 The definition of `Time`, `Duration`, `Url`, and `BatchMode` follow those in
 {{!DAP}}.
 
+## Extensions {#taskbind-extensions}
+
+The `TaskConfig` structure includes a list of extensions. In general,
+extensions can be used to bind additional, application-specific information to
+the task. For example, an extension might be used to encode the identity of the
+Collector. (Only the Aggregators are identified in `TaskConfig`.)
+
+Each extension is structured as follows:
+
+~~~ tls-presentation
+struct {
+  ExtensionType extension_type;
+  opaque extension_data<0..2^16-1>;
+} Extension;
+
+enum {
+  reserved(0),
+  (2^16-1)
+} ExtensionType;
+~~~
+
+The `extension_type` identifies the extension and `extension_data` is
+structured as specified by the extension.
+
+Extensions are treated as mandatory-to-implement in the protocol described in
+{{taskprov}}. In particular protocols participants MUST opt-out of tasks
+containing unrecognized extensions. See {{provisioning-a-task}}.
```

Comment:
The question I have is how codepoints here relate to codepoints in DAP, if that makes sense.

If I define an extension 0xAAAA here does that match the 0xAAAA codepoint in DAP?

Reply:
+1

Reply:
I think this makes sense, do we want to add some text to explain this strategy. I can see this same question coming up in the future.

Reply:
Yup, will do.

Reply:
Done.